projects/canopy/src/lib/carousel/carousel.component.html

```diff
@@ -0,0 +1,55 @@
+<section class="lg-carousel">
+  <lg-heading [level]="headingLevel" class="lg-carousel__heading lg-visually-hidden">{{
+    carouselTitle
```

Comment:
I wonder if this should be content projected as opposed to a string? Content projection would keep it consistent with the other components and allows a bit more flexibility.

Reply:
I think this only needs to be a string as it will just populate the heading describing the carousel for customers with screen readers

Reply:
Resolving this as we discussed this over a call